### DISCUSSION TITLE
Need way to embed lucid chart

### DISCUSSION THREAD

**Discussion by benzitney** [GitbookIO/community#280] [Feature Requests]
Currently there's no way to embed lucid chart assets in Gitbook. The only way to embed a lucid chart asset is if you include a full HTML embed code. Using a simple URL doesn't work.

So we would either need:
A. A way to paste in the embed code — rather than only pasting in a embed URL
B. An integration with Lucid that allows an asset to be embedded with only the use of a URL

**Comment by addisonschultz** [comment]
Hey @benzitney! Thanks for pointing this out - this should definitely be possible to be built using our Integration Platform: https://developer.gitbook.com/

You'd need to build an integration that unfurls the URL to display it in GitBook. We even have a guide on that here! 

https://developer.gitbook.com/getting-started/guides/create-a-custom-unfurl-action-for-your-integration

**Comment by benzitney** [comment]
Thank you @addisonschultz  but to be frank I don't even know where to start with those instructions. I'm not an web/app developer so for example I don't even know where to start with instructions like this https://developer.gitbook.com/cli/quickstart and I don't have the time to go down the rabbit hole to learn these skills just to get an iframe to display

**Comment by addisonschultz** [comment]
@benzitney totally understandable. That being said, we just launched a new integration with Lucid! I think this might work for what you're looking for 🥳 

https://www.gitbook.com/integrations/lucid

Let me know how it works out for you!
